src/embeddedWebserver.h

```diff
@@ -161,15 +161,15 @@ void paramToJson(String name, editable_t &e, DynamicJsonDocument &doc) {
 // Use libraries for the webinterface from the internet (0) or from the local filesystem (1). 0 has slightly faster load times
 #define NOINTERNET 1
 
-//hash strings at compile time to use in switch statement
-//(from https://stackoverflow.com/questions/2111667/compile-time-string-hashing)
+// hash strings at compile time to use in switch statement
+// (from https://stackoverflow.com/questions/2111667/compile-time-string-hashing)
 constexpr unsigned int str2int(const char* str, int h = 0) {
     return !str[h] ? 5381 : (str2int(str, h+1) * 33) ^ str[h];
 }
 
 String getHeader(String varName) {
-    //TODO: actually put the references libs on local file system again (only when using ESP32 which has more flash mem,
-    //but also make sure web server can handle this many concurrent requests (might crash)
+    // TODO: actually put the references libs on local file system again (only when using ESP32 which has more flash mem,
```

Comment:
I think this comment can go, all files are in the filesystem.
Btw, since when is NOINTERNET 1 by default?

Reply:
Wasn't me! ;D

Reply:
I guess it's not a bad default for consistency, just slower